diff --git a/swig/CMakeLists.txt b/swig/CMakeLists.txt
index 93f4b63ab..f16051362 100644
--- a/swig/CMakeLists.txt
+++ b/swig/CMakeLists.txt
@@ -97,7 +97,7 @@ include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/TimeHandling )
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/Utilities )
 include_directories(AFTER SYSTEM ${PROJECT_BINARY_DIR})
 
-set(TK_EXTRA_FLAGS -DGNSSTK_EXPORT -DSWIG_TYPE_TABLE=gnsstk)
+set(TK_EXTRA_FLAGS -DGNSSTK_EXPORT= -DSWIG_TYPE_TABLE=gnsstk)
 
 set( CLOCKMODEL_MODULE "ClockModel" )
 set(SWIG_MODULE_${CLOCKMODEL_MODULE}_EXTRA_FLAGS ${TK_EXTRA_FLAGS})
diff --git a/swig/Geomatics/Geomatics.i b/swig/Geomatics/Geomatics.i
index a4e21667c..1b5f70c1b 100644
--- a/swig/Geomatics/Geomatics.i
+++ b/swig/Geomatics/Geomatics.i
@@ -89,16 +89,19 @@ namespace std
 %feature("flatnested", "");
 %include "AntexHeader.hpp"
 %include "AntexStream.hpp"
+%apply std::string &OUTPUT { std::string &name };
 %include "AntennaStore.hpp"
+%clear std::string &name;
 %include "EphTime.hpp"
 %include "AtmLoadTides.hpp"
 %include "CubicSpline.hpp"
 /* %include "SatPass.hpp" */
 /* %include "DiscCorr.hpp" */
 %include "IERSConvention.hpp"
-/* %include "EarthOrientation.hpp" */
-/* %include "EOPPrediction.hpp" */
-/* %include "EOPStore.hpp" */
+%ignore gnsstk::EarthOrientation::ECEFtoJ2000;
+%include "EarthOrientation.hpp"
+%include "EOPPrediction.hpp"
+%include "EOPStore.hpp"
 %include "RobustStats.hpp"
 %include "StatsFilterHit.hpp"
 %include "FDiffFilter.hpp"
@@ -126,7 +129,7 @@ namespace std
 %include "SolarSystemEphemeris.hpp"
 %include "SolidEarthTides.hpp"
 %include "SunEarthSatGeometry.i"
-/* %include "SolarSystem.hpp" */
+%include "SolarSystem.hpp"
 /* %include "PreciseRange.hpp" */
 /* %include "Rinex3ObsFileLoader.hpp" */
 /* %include "SRIleastSquares.hpp" */
diff --git a/swig/gnsstk_swig.hpp b/swig/gnsstk_swig.hpp
index 9e313ba24..1988b476d 100644
--- a/swig/gnsstk_swig.hpp
+++ b/swig/gnsstk_swig.hpp
@@ -175,9 +175,9 @@
 /* #include "DiscCorr.hpp" */
 #include "ENUUtil.hpp"
 #include "IERSConvention.hpp"
-/* #include "EarthOrientation.hpp" */
-/* #include "EOPPrediction.hpp" */
-/* #include "EOPStore.hpp" */
+#include "EarthOrientation.hpp"
+#include "EOPPrediction.hpp"
+#include "EOPStore.hpp"
 #include "EngAlmanac.hpp"
 #include "EngEphemeris.hpp"
 #include "NavValidityType.hpp"
@@ -386,7 +386,7 @@
 #include "SolarSystemEphemeris.hpp"
 #include "SolidEarthTides.hpp"
 #include "SunEarthSatGeometry.hpp"
-/* #include "SolarSystem.hpp" */
+#include "SolarSystem.hpp"
 /* #include "PreciseRange.hpp" */
 #include "RACRotation.hpp"
 /* #include "RTFileFrame.hpp" */
diff --git a/swig/gnsstk_swig.i b/swig/gnsstk_swig.i
index 008b6eaf7..f56a26e48 100644
--- a/swig/gnsstk_swig.i
+++ b/swig/gnsstk_swig.i
@@ -225,9 +225,10 @@
 /* %import(module="gnsstk") "DiscCorr.hpp" */
 %import(module="gnsstk") "ENUUtil.hpp"
 %import(module="gnsstk") "IERSConvention.hpp"
-/* %import(module="gnsstk") "EarthOrientation.hpp" */
-/* %import(module="gnsstk") "EOPPrediction.hpp" */
-/* %import(module="gnsstk") "EOPStore.hpp" */
+%ignore gnsstk::EarthOrientation::ECEFtoJ2000;
+%import(module="gnsstk") "EarthOrientation.hpp"
+%import(module="gnsstk") "EOPPrediction.hpp" 
+%import(module="gnsstk") "EOPStore.hpp" 
 %import(module="gnsstk") "EngAlmanac.hpp"
 %import(module="gnsstk") "EngEphemeris.hpp"
 %import(module="gnsstk") "NavValidityType.hpp"
@@ -458,7 +459,7 @@
 %import(module="gnsstk") "SolarSystemEphemeris.hpp"
 %import(module="gnsstk") "SunEarthSatGeometry.i"
 %import(module="gnsstk") "SolidEarthTides.hpp"
-/* %import(module="gnsstk") "SolarSystem.hpp" */
+%import(module="gnsstk") "SolarSystem.hpp" 
 /* %import(module="gnsstk") "PreciseRange.hpp" */
 %import(module="gnsstk") "RACRotation.hpp"
 %import(module="gnsstk") "RNDouble.hpp"
